src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -64,7 +64,7 @@ private SyntaxNode GetFixedNode(dynamic fixedRoot)
             await TestMissingAsync(@"class C { int [||]x = 42; void M() { System.Console.WriteLine(x); } }");
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/12838"), Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)]
         public async Task WithRefInitializer1()
         {
             await TestMissingAsync(@"
```

Comment:
There is an open issue to address this now.

---
In reply to: [73058619](https://github.com/dotnet/roslyn/pull/12841#discussion_r73058619) [](ancestors = 73058619)